usage_examples/rel_openai/fewshot.cfg

```diff
@@ -0,0 +1,26 @@
+[paths]
+examples = null
+
+[nlp]
+lang = "en"
+pipeline = ["ner", "llm_rel"]
+
+[components]
+
+[components.ner]
+source = "en_core_web_md"
```

Comment:
This `source` does not include the vectors. If you use `spacy assemble` or `initialize` you'll get a warning, but the way you're running things with these `util` methods (that aren't really meant to be used like this) things aren't going through the normal spacy workflow+checks.

Reply:
> @bdura [This](https://github.com/explosion/spacy-llm/actions/runs/5065991860/jobs/9095218237#step:5:195) is the warning I mean. Can you reproduce this locally?

@rmitsch I imagine this is linked?

@adrianeboyd what's the proper way to use another pipeline's component through the configuration?

Reply:
> I imagine this is linked?

I'd think so.